packages/p2p/src/utils/date-time.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Gets the formatted date string in the format "DD MMM YYYY HH:mm:ss". e.g.: "01 Jan 1970 21:01:11"
+ * or "MMM DD YYYY HH:mm:ss" for local time. e.g.: "Jan 01 1970 21:01:11" or without seconds if
+ * has_seconds is false. e.g.: "01 Jan 1970 21:01" or "Jan 01 1970 21:01".
+ *
+ * @param {Date} date_obj - The date object to format.
+ * @param {boolean} is_local - Whether to use local time or UTC time.
+ * @param {boolean} has_seconds - Whether to include seconds in the time.
+ * @returns {String} The formatted date string.
+ */
+export const getFormattedDateString = (date_obj: Date, is_local = false, has_seconds = false): string => {
+    const date_string = is_local ? date_obj.toString().split(' ') : date_obj.toUTCString().split(' ');
+    const [, day, month, year, time] = date_string;
```

Comment:
```suggestion
    const [day, month, year, time] = date_string;
```

:thinking:

Reply:
yeah this was from the previous code, let me check if this can be removed

Reply:
Thanks Ameerul :pray:

Reply:
ok so if i remove the comma at the front, it will cause a syntax error and the code will not compile. The comma is used to separate the date and time strings in the returned formatted date string so we need to leave the comma there

Reply:
I see thanks for checking

Reply:
```suggestion
    const [_, day, month, year, time] = date_string;
```

maybe like this? so we know there is a value that we dont use there

Reply:
yup thatll be better 👍